tests/integration/test_tf_integration.py

```diff
@@ -249,7 +248,7 @@ def test_eng_run_state_mean_photon(self, setup_eng, tol):
         assert np.allclose(var, ref_var, atol=tol, rtol=0.0)
 
 
-class TestTwoModeSymbolic:
+class TestTwoModes:
```

Comment:
How come this was changed?

Reply:
The name didn't fit anymore now that default is eager